pkg/services/ngalert/notifier/multiorg_alertmanager_test.go

```diff
@@ -285,9 +285,9 @@ func TestMultiOrgAlertmanager_AlertmanagerFor(t *testing.T) {
 	{
 		am, err := mam.AlertmanagerFor(2)
 		require.NoError(t, err)
-		require.Equal(t, *am.GetStatus().VersionInfo.Version, "N/A")
-		require.Equal(t, am.orgID, int64(2))
-		require.NotNil(t, am.Base.ConfigHash())
+		require.Equal(t, "N/A", *am.GetStatus().VersionInfo.Version)
```

Comment:
This is only a small detail, the `require.Equal()` function requires the expected value to be second and the actual value to be third.